modules/basics/src/main/java/com/opengamma/basics/BuySell.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.basics;
+
+import org.joda.convert.FromString;
+import org.joda.convert.ToString;
+
+import com.google.common.base.CaseFormat;
+import com.opengamma.collect.ArgChecker;
+
+/**
+ * Flag indicating whether a trade is "buy" or "sell".
```

Comment:
should probably specify for whom it is buy or sell - as for someone else it will be the opposite